File: algo/warmup/js/miniMaxSum.js
function miniMaxSum(arr) {
    let min = Number.MAX_SAFE_INTEGER;
    let max = Number.MIN_SAFE_INTEGER;
    const sum = arr.reduce((prev, curr) => {
        if (curr > max) { max = curr; }
        if (curr < min) { min = curr; }
        return prev + curr;
    }, 0);
    console.log(sum - max, sum - min);
}

File: algo/warmup/cpp/miniMaxSum.cpp
void miniMaxSum(vector<int> arr)
{
    vector<int>::iterator It;
    long int sum = 0;
    int min = INT_MAX;
    int max = INT_MIN;
    for (It = arr.begin(); It != arr.end(); It++)
    {
        sum += *It;
        if (*It < min)
        {
            min = *It;
        }
        if (*It > max)
        {
            max = *It;
        }
    }
    cout << sum - max << " " << sum - min;
}
